lms/templates/instructor/instructor_dashboard_2/make_invoice_transaction_modal.html

```diff
@@ -0,0 +1,119 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.core.urlresolvers import reverse %>
+<%page args="section_data"/>
+<section id="make_invoice_transaction_modal" class="modal" role="dialog" tabindex="-1" aria-label="${_('Add Coupon')}">
+    <div class="inner-wrapper">
+        <button class="close-modal">
+            <i class="icon fa fa-remove"></i>
+      <span class="sr">
+        ## Translators: this is a control to allow users to exit out of this modal interface (a menu or piece of UI that takes the full focus of the screen)
+        ${_('Close')}
+      </span>
+        </button>
+
+        <div id="coupon-content">
+            <header>
+                <h2>${_("Add invoice payments and refunds")}</h2>
+            </header>
+            <h3 class="error-msgs" id="error-msgs-invoice"></h3>
+
+            <div class="instructions">
+                <p>
+                    ${_("Please enter valid credentials")}</p>
```

Comment:
I think @stephensanchez was referring to the string.  "credentials" usually mean username / password, which doesn't make sense in this context.